examples/cookbook/app/network-requests/__tests__/PhoneBook.test.tsx

```diff
@@ -0,0 +1,119 @@
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react-native';
+import React from 'react';
+import axios from 'axios';
+import PhoneBook from '../PhoneBook';
+import { User } from '../types';
+
+jest.mock('axios');
+
+jest.setTimeout(10000);
```

Comment:
When this test suit runs in CI, the default 5s is not sufficient.
Locally it's a matter of 0.91 s.

Any idea what might be causing this on CI? @mdjastrzebski?